stl/inc/xutility

```diff
@@ -162,6 +162,13 @@ _NODISCARD _CONSTEXPR_BIT_CAST _To _Bit_cast(const _From& _Val) noexcept {
 #endif // ^^^ no workaround ^^^
 }
 
+template <class _Ty>
+_NODISCARD _Ty _Fake_decay_copy(_Ty) noexcept;
+// _Fake_decay_copy<T>(E):
+// (1) has type T [decay_t<decltype((E))> if T is deduced],
+// (2) is well-formed if and only if E is implicitly convertible to T and T is destructible, and
+// (3) is non-throwing if and only if both conversion from decltype((E)) to T and destruction of T are non-throwing.

```

Comment:
Why is this being moved up from right line 1683, and extracted out of `#ifdef __cpp_lib_concepts`, when nothing else in the PR mentions it? (Looking at the commit history, it appears that this is a relic of intermediate steps - it was necessary to be moved up at one point, but no longer.)